packages/dmn-js-drd/test/spec/features/auto-place/DmnAutoPlaceSpec.js

```diff
@@ -3,10 +3,10 @@ import {
   inject
 } from 'test/TestHelper';
 
-import autoPlaceModule from 'lib/features/auto-place';
-import coreModule from 'lib/core';
-import labelEditingModule from 'lib/features/label-editing';
-import modelingModule from 'lib/features/modeling';

```

Comment:
Sorry if I don't have enough context here, but is this intended?

Reply:
Ah sorry, I see: https://github.com/bpmn-io/dmn-js/pull/684/commits/f26351a988ea9d89d618fb9540cf83d275972b2f

Interesting though!

Reply:
I hope that this will improve the coverage metrics a bit, but it's probably not the only problem with the coverage in this project.